packages/integration-tests/api/granules.js

```diff
@@ -162,7 +162,20 @@ async function removePublishedGranule({ prefix, granuleId }) {
   return deleteGranule({ prefix, granuleId });
 }
 
+const createGranule = (prefix, granule) =>
```

Comment:
I discussed this with Marc offline, but the API does not actually support this type of action, and I am of the opinion that for granules and other records created during ingest (e.g. PDRs, executions, etc.) we want to use the typical workflow to create those records, i.e. invoking the `cw-sf-event-to-db-records` lambda with a StepFunction CW Event wrapper.

Reply:
What do you mean that the API does not support this type of action? It's just a POST to the /granules endpoint?

Also, I thought that for other workflows that want to create/update records during ingest, they would use the new SQS queue in front of reporting that you are creating?

Reply:
Agree with @nemreid , even if this would make my integration test work simpler.      This might be worth a quick meet/goto to hash out if we're not on the same page re: this.

Reply:
@markdboyd 

>What do you mean that the API does not support this type of action? It's just a POST to the /granules endpoint?

And `/granules` does not support POST requests. PUT requests only supports reingest, removeFromCMR, move and applyWorkflow. There is no function to create a granule on the API.

> Also, I thought that for other workflows that want to create/update records during ingest, they would use the new SQS queue in front of reporting that you are creating?

Yes, but since my PR is not ready for review I didn't want to impose waiting on me as a blocker for this work. For the purpose of granules, invoking the lambda directly would still take the same input and get the same result.

Reply:
> And /granules does not support POST requests

Well then. I just assumed that it did